spec/redmine/redmine_issue_spec.rb

```diff
@@ -0,0 +1,59 @@
+require './spec/spec_helper'
+
+describe '[Issue]', :redmine, :issue do
+  user = RedmineUser.new
+  project = RedmineProject.new
```

Comment:
I was trying to find a solution here, so the 'prerequisite' objects could be available in:
- nested example groups and examples
- all relative lets, befores and afters
- (!) as an argument for nested and relative `include_context` and `include_examples` (which is a main issue now)

And I can see two options:
1) I'm structuring specs wrong so it makes me start looking for a solution here
2) Or I can use instance variables in `before` hooks like

```Ruby
describe '[Issue]' do
  before :all do
    @user = RedmineUser.new
    @project = RedmineProject.new
  end
  context 'I am nested context' do
    include_examples 'Register as a', @user
    include_examples 'Create a new project', @project
    # nested contexts and examples which could use these variables unlimitly
  end
end
```
https://www.evernote.com/shard/s19/sh/9e13bd34-5db5-49b1-aa5c-03fc8310d88e/bb656f5793aee5591a36c87b5ff7ec8a

Hope you wouldn't mind if I come to your table tomorrow :)